src/towncrier/_writer.py

```diff
@@ -24,9 +21,7 @@ def append_to_newsfile(directory, filename, name_and_version, content):
         with open(news_file, "r") as f:
             existing_content = f.read()
 
-    existing_content = existing_content.split(TOWNCRIER_START)
-
-    top_line = name_and_version + "\n" + "=" * len(name_and_version) + "\n"
+    existing_content = existing_content.split(start_line)
```

Comment:
I'm guessing we want to make this `.split(start_line, 1)` in case it's something that shows up multiple times...